frontend/src/hocs/withScreencast/components/VideoRecordedDialog/styled-components.ts

```diff
@@ -0,0 +1,49 @@
+import styled from "@emotion/styled"
+
+export const StyledVideo = styled.video(({ theme }) => ({
+  width: theme.sizes.full,
+  borderRadius: theme.radii.md,
+}))
+
+export const StyledDialogContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: theme.sizes.full,
+}))
+
+export const StyledRow = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  paddingTop: theme.spacing.md,
+  paddingBottom: theme.spacing.md,
+}))
+
+export const StyledFirstColumn = styled.div(({ theme }) => ({
+  paddingRight: theme.spacing.lg,
+  textAlign: "right",
+  color: theme.colors.gray,
+  fontWeight: theme.fontWeights.bold,
+  width: "6em",
+}))
+
+export const StyledSecondColumn = styled.div(({ theme }) => ({
+  flex: 1,
+  paddingRight: theme.spacing.lg,
+}))
+
+export const StyledThirdColumn = styled.div(({ theme }) => ({
+  width: "6em",
+  [`@media (max-width: ${theme.breakpoints.sm})`]: {
+    width: 0,
+  },
+}))
```

Comment:
Couldn't this be margin-right on the 2nd column?

Reply:
Nice! Simplifies it more. I'll make the change.